_includes/footer.html

```diff
@@ -3,6 +3,31 @@
 
         <div class="footer-pre">
 
+            <a class="btn btn__secondary footer_top-button js-return-to-top" href="#"> Back to top <span class="cf-icon cf-icon-arrow-up"></span></a>
```

Comment:
Minor... let's stack this line so it isn't so long... aim to fit it within 80 characters.
Something like:

```
<a href="#" class="btn btn__secondary footer_top-button js-return-to-top"> 
    Back to top <span class="cf-icon cf-icon-arrow-up"></span>
</a>
```

…or…

```
<a href="#"
   class="btn
          btn__secondary
          footer_top-button
          js-return-to-top
          "> 
    Back to top <span class="cf-icon cf-icon-arrow-up"></span>
</a>
```